File: leetcode-submissions/1684_Count_the_Number_of_Consistent_Strings.cpp
class Solution {
public:
    int countConsistentStrings(string allowed, vector<string>& words) {
        set<int>unique(allowed.begin(),allowed.end());
        int ans=0;
        for(int i=0;i<words.size();i++){
            string word=words[i];
            bool flag=true;
            for(char ch:word){
                if(!unique.count(ch)){
                    flag=false;
                    break;
                }
            }
            if(flag){
                ans+=1;
            }
            
        }
        return ans;
    }
};

File: leetcode-submissions/1684_Count_the_Number_of_Consistent_Strings.ts
function countConsistentStrings(allowed: string, words: string[]): number {
let ans:number=0;
const unique=new Set(allowed);
for(const word of words){
    const wordArr=word.split('');
    let flag:boolean=true;
    for(let i=0;i<wordArr.length;i++){
        if(!unique.has(wordArr[i])){
            flag=false;
            break;

        }
    }
    if(flag){
        ans+=1;
    }

}
return ans;
};